toolkits/global/packages/global-popup/js/popup.js

```diff
@@ -110,8 +104,8 @@ const Popup = class {
 		// calc space above trigger
 		const spaceAbove = offset.top - this._content.offsetHeight - arrowHeight;
 
-		const abovePosition = 0 - this._content.offsetHeight - arrowHeight;
-		const belowPosition = 0 + triggerMetrics.height + arrowHeight;
+		const abovePosition = offset.top - this._content.offsetHeight - arrowHeight;
```

Comment:
👋 I presume it's no coincidence that this is the same calculation as `spaceAbove`. Does it make the `spaceAbove` var redundant?